backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -87,4 +81,12 @@ public async Task<List<ProjectResponseDto>> GetAllUserProjectsAsync()
 
         return _mapper.Map<List<ProjectResponseDto>>(userProjects)!;
     }
-}
+
+    private void ValidateProject(Project? entity)
+    {
+        if (entity is null || entity.CreatedBy != _userIdGetter.GetCurrentUserId())
+        {
+            throw new EntityNotFoundException(nameof(Project));
```

Comment:
it seems when we have this `entity.CreatedBy != _userIdGetter.GetCurrentUserId()` condition we should throw another Exception (like `NoPermissionException`)

Reply:
if we throw this exception, we let user know about existing of project with such id (I think useless information), user does not need to know much